test/common/deployment_types.go

```diff
@@ -205,22 +206,22 @@ func TestDeploymentExpectedReplicas(t *testing.T, ctx *TestingContext) {
 			}
 
 			if deployment.Status.Replicas != product.ExpectedReplicas {
-				t.Errorf("Deployment %s in namespace %s doens't match the number of expected replicas %s/%s",
+				t.Errorf("Deployment %s in namespace %s doesn't match the number of expected replicas. Replicas: %s / Expected Replicas: %s",
 					product.Name,
 					namespace.Name,
-					string(deployment.Status.Replicas),
-					string(product.ExpectedReplicas),
+					strconv.Itoa(int(deployment.Status.Replicas)),
+					strconv.Itoa(int(product.ExpectedReplicas)),
 				)
 				continue
 			}
 
 			// Verify that the expected replicas are also available, means they are up and running and consumable by users
 			if deployment.Status.AvailableReplicas != product.ExpectedReplicas {
-				t.Errorf("Deployment %s in namespace %s doens't match the number of expected available replicas %s/%s",
+				t.Errorf("Deployment %s in namespace %s doesn't match the number of expected available replicas. Available Replicas: %s / Expected Replicas: %s",
 					product.Name,
 					namespace.Name,
-					string(deployment.Status.AvailableReplicas),
-					string(product.ExpectedReplicas),
+					strconv.Itoa(int(deployment.Status.AvailableReplicas)),
```

Comment:
@davidffrench using `%v` instead of `%s` and then not casting the avaible and expected replicas to strings also works

Reply:
Thanks @pb82 . Much cleaner. Updated now